### PR TITLE
Add support for 2024.3 symbol dictionaries

### PR DESCRIPTION
**Pull request by LeonarddeR**
Note, this also fixes translation of braille table display names in localized manifests.

### CODE REVIEW

**Comment by nvdaes** [comment]
Thanks so much @LeonarddeR . I was thinking about requesting this on the add-ons mailing list time ago,but then I started doing other things and I never requested it.
I use dictionaries in an add-on named Perky, for Perky  Duck, a program to learn braille. I'm not sure if this will be appropriate for the add-on, but I'll study it.

File: buildVars.py [hunk 1]
```diff
@@ -87,3 +87,12 @@ def _(arg):
 # output (shown in output table list),
 # input (shown in input table list).
 brailleTables = {}
+
+# Custom speech symbol dictionaries
+# Symbol dictionary files reside in the locale folder, e.g. `locale\en`, and are named `symbols-<name>.dic`.
+# If your add-on includes custom speech symbol dictionaries (most will not), fill out this dictionary.
+# Each key is the name of the dictionary,
+# with keys inside recording the following attributes:
+# displayName (name of the table shown to users and translatable),
```
Comment (nvdaes) [inline buildVars.py:96]:
```suggestion
# displayName (name of the speech dictionary  shown to users and translatable),
```

File: readme.md [hunk 2]
```diff
@@ -115,6 +118,15 @@ In addition to the core manifest data, custom add-on information can be specifie
 
 Note: you must fill out this dictionary if at least one custom braille table is included in the add-on. If not, leave the dictionary empty.
 
+###### SPeech symbol dictionaries
```
Comment (nvdaes) [inline readme.md:121]:
```suggestion
###### Speech symbol dictionaries
```

**Review by nvdaes — CHANGES_REQUESTED** [review]

**Review by josephsl — APPROVED** [review]
Thank you for your contribution (and good luck in the next adventure).

**Pull request merged**